src/modules/profile/actions.ts

```diff
@@ -14,3 +15,13 @@ export const loadProfileFailure = (address: string, error: string) => action(LOA
 export type LoadProfileRequestAction = ReturnType<typeof loadProfileRequest>
 export type LoadProfileSuccessAction = ReturnType<typeof loadProfileSuccess>
 export type LoadProfileFailureAction = ReturnType<typeof loadProfileFailure>
+
+// Change Profile
+export const CHANGE_PROFILE = 'Change Profile'
+
+export const changeProfile = (
+  address: string,
+  profile: Partial<Profile> & { avatars: Partial<Avatar>[]; content?: any[]; metadata?: any[] }
```

Comment:
Maybe we can name this type and add it to `/types.ts`.

@cazala  Does this exist elsewhere?

Reply:
this should be just `(address: string, profile: Profile)` no need to send the whole `entity` or a partial profile

Reply:
you're right! after a too many comings and goings with the Partial<Profile> is not needed anymore. Fixed!